test/models/test_data_services.rb

```diff
@@ -321,22 +321,30 @@ def test_it_can_be_droby_marshalled_and_unmarshalled
         end
     end
 
-    describe "#resolve" do
+    describe "#bind" do
         it "returns the value of try_resolve is non-nil" do
             srv_m = DataService.new_submodel
             flexmock(srv_m).should_receive(:try_resolve).with(task = flexmock).and_return(obj = flexmock)
```

Comment:
should be better if everything was renamed to bind, including try_resolve => try_bind

Reply:
This was a very astute observation, and a big rabbit hole.

I've update the commit (and the pull request) to clean up the usage of 'resolve' and 'bind' within the whole of Syskit. I spotted an issue with InstanceRequirements, and decided to leave it as it was as changing the behavior of IR#bind would IMO be far beyond the scope of this change (but will still need to be done)

Reply:
I've updated the commit and PR messages to reflect the problem with InstanceRequirements